content/id/docs/concepts/services-networking/connect-applications-service.md

```diff
@@ -0,0 +1,366 @@
+```
+Sekarang buat *secrets* menggunakan file tersebut:
+```shell
+kubectl apply -f nginxsecrets.yaml
+kubectl get secrets
+```
+```
+NAME                  TYPE                                  DATA      AGE
+default-token-il9rc   kubernetes.io/service-account-token   1         1d
+nginxsecret           Opaque                                2         1m
+```
+
+Sekarang modifikasi replika *nginx* untuk menjalankan server *https* menggunakan sertifikat didalam *secret* dan *Service* untuk mengekspos semua *port* (80 dan 443):
+
+{{< codenew file="service/networking/nginx-secure-app.yaml" >}}
+
+Berikut catatan penting tentang manifes *nginx-secure-app*:
+
+- Didalam file tersebut, ada spesifikasi *Deployment* dan *Service*
+- ada [server nginx](https://github.com/kubernetes/examples/tree/{{< param "githubbranch" >}}/staging/https-nginx/default.conf) yang melayani trafik *HTTP* di *port* 80 dan trafik *HTTPS* di *port* 443, dan *Service nginx* yang mengekspos kedua *port* tersebut.
+- Setiap kontainer mempunyai akses ke *key* melalui *volume* yang di *mount* pada `/etc/nginx/ssl`. Ini adalah konfigurasi sebelum server *nginx* dijalankan.
+
+```shell
+kubectl delete deployments,svc my-nginx; kubectl create -f ./nginx-secure-app.yaml
+```
+
+Pada titik ini, kamu dapat berkomunikasi dengan server *nginx* dari *node* manapun.
+
+```shell
+kubectl get pods -o yaml | grep -i podip
+    podIP: 10.244.3.5
+node $ curl -k https://10.244.3.5
+...
+<h1>Welcome to nginx!</h1>
+```
+
+Perlu dicatat bahwa kita menggunakan parameter `-k` saat menggunakan *curl*, ini karena kita tidak tau apapun tentang *Pod* yang menjalankan *nginx* saat pembuatan seritifikat, jadi kita harus memberitahu *curl* untuk mengabaikan ketidakcocokan *CName*. Dengan membuat *Service*, kita menghubungkan *CName* yang digunakan pada sertifikat dengan nama pada *DNS* yang digunakan *Pod*. Lakukan pengujian dari sebuah *Pod* (*secret* yang sama digunakan untuk agar mudah, *Pod* tersebut hanya membutuhkan *nginx.crt* untuk mengakses *Service*)
```

Comment:
> (*secret* yang sama digunakan untuk agar mudah, *Pod* tersebut hanya membutuhkan *nginx.crt* untuk mengakses *Service*)

this sentence doesn't make sense, probably missing translation?

Reply:
I am translating from `the same secret is being reused for simplicity, the pod only needs nginx.crt to access the Service`.

Reply:
let's translate this into:
```
*secret* yang sama digunakan kembali untuk penyederhanaan, ...
```